mods/ts/rules/defaults.yaml

```diff
@@ -1,3 +1,14 @@
+^ExistsInWorld:
+	AppearsOnRadar:
+	UpdatesPlayerStatistics:
+	CombatDebugOverlay:
+	DrawLineToTarget:
```

Comment:
This looks like an overkill, considering not even half of the actors that inherit this will need target lines.